### DISCUSSION TITLE
Abundance and occupancy remaining at 0 despite SpDist file with occupant cells, TransMatrix and landscape code matches

### DISCUSSION THREAD

**Discussion by Markus-Shiweda** [RangeShifter/RangeshiftR-tutorials#45] [Q&A]
Hi @annekathleenmalchow and all

I have  realised that `Abundance and occupancy` is remaining at 0 despite SpDist file with occupant cells, and landscape codes that are = 8 (1,2,3,4,5,6,7,8) matches carrying capacity : carrycap1 <- c(5, 5, 4, 1, 0, 1, 2, 3). Both have the same resolution 300m x 300m. 

I am using RangeShiftR package.
This was slightly discussed before but I seem to have sorted everything out. 

What may be the potential cause?

**Comment by JetteReeg** [comment]
Hi, it would be great if you could provide a minimal working example, so I can have a look at the issue.
